cypress/e2e/1-getting-started/todo.cy.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
```

Comment:
probably add these to the `.gitignore` if you want to keep them on the local filesystem

Reply:
Removed the examples from the project.